cypress/integration/rollups_spec.js

```diff
@@ -21,6 +21,9 @@ describe("Rollups", () => {
     // // Click on "Sample data" tab
     // cy.contains("Add data").click({ force: true });
```

Comment:
Do we need the duplicate code here?

Reply:
This is a bit of a flaky test, sometimes needs the commented code to pass, sometimes doesn't.

Reply:
oh I see, yes I'll get rid of the commented, since it is being used.

Reply:
Done.